server/src/FaceRest.h

```diff
@@ -82,6 +82,11 @@ class FaceRest : public FaceBase {
 	static void handlerLogin(ResourceHandler *job);
 	static void handlerLogout(ResourceHandler *job);
 
+	int onCaughtException(const std::exception &e)
```

Comment:
Please add the override keyword as

int onCaughtException(const std::exception &e) override